app/views/layouts/application.html.haml

```diff
@@ -1,58 +1,70 @@
+<html lang="en" class="ie7 no-js">    <![endif]-->
+<!--[if IE 8 ]>
+<html lang="en" class="ie8 no-js">    <![endif]-->
+<!--[if IE 9 ]>
+<html lang="en" class="ie9 no-js">    <![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!-->
+<html lang="en" class="no-js"><!--<![endif]-->
 
-%html.js{lang: "en-au"}
-  = render 'application/head',  title: (content_for?(:title) ? "#{yield(:title)} | GOV.AU" : 'GOV.AU' )
-  %body
-    %div.skip-to
-      %a{href: "#content"}
-        Skip to main content
-    %header{role: 'banner'}
-      = render 'application/signed_in_nav' if user_signed_in?
-      %section.prerelease-govau--header
-        - if content_for?(:header)
-          = yield(:header)
-        - else
-          .wrapper
-            %a.govau--logo{href: "/"}
-              = image_tag 'icons/logo-govau-full-2x.png', width: '270'
-              .brandbar.default
-                .b1
-                .b2
-            = #render partial: 'feedback_and_menu'
+= render 'application/head',  title: (content_for?(:title) ? "#{yield(:title)} | GOV.AU" : 'GOV.AU' )
+%body
+  %div.skip-to
+    %a{href: "#content"}
+      Skip to main content
+  %header{role: 'banner'}
+    = render 'application/signed_in_nav' if user_signed_in?
+    %section.prerelease-govau--header
+      - if content_for?(:header)
+        = yield(:header)
+      - else
+        .wrapper
+          %a.govau--logo{href: "/"}
+            = image_tag 'icons/logo-govau-full-2x.png', width: '270'
+            .brandbar.default
+              .b1
+              .b2
+          = #render partial: 'feedback_and_menu'
 
-          %nav.breadcrumbs--inverted{'aria-label' => 'breadcrumb'}
-            .wrapper
-              = breadcrumbs style: :ul, class: nil
+      %nav.breadcrumbs--inverted{'aria-label' => 'breadcrumb'}
+        .wrapper
+          = breadcrumbs style: :ul, class: nil
 
-    = yield(:after_header)
+= yield(:after_header)
 
-    %main
-      - if content_for?(:navbar)
-        = yield :navbar
-      = render 'application/alerts'
-      = yield
-    %footer{role: "contentinfo"}
-      .wrapper
-        %section
-          %figure
-            .footer-logo
-              = image_tag 'coat-of-arms.png', alt: 'Australian Government Coat of Arms'
-          .footer-links
-            %nav
-              %ul
-                %li
-                  %a{href: 'javascript:void(0)'} About
-                %li
-                  %a{href: 'javascript:void(0)'} Accessibility
-                %li
-                  %a{href: 'javascript:void(0)'} Copyright
-                %li
-                  %a{href: 'javascript:void(0)'} Help
-                %li
-                  %a{href: 'javascript:void(0)'} Site map
-                %li
-                  %a{href: 'javascript:void(0)'} Terms of use
-            %p
-              &copy; Commonwealth of Australia
-              %br
-              %a{href:'https://github.com/AusDTO/gov-au-ui-kit/blob/master/LICENSE.md'} Licensed under the MIT License.
+%main
```

Comment:
`main` is now after the `body` tag rather than inside it. This seems odd. Similarly the `footer`.